postgres/src/test/java/org/jdbi/v3/postgres/TestEnumSets.java

```diff
@@ -148,6 +149,26 @@ public void testAmountPlatforms() {
         assertThat(amount).isEqualTo(3);
     }
 
+    @Test
+    public void throwsOnNonBitChars() {
+        db.getHandle().useTransaction(handle -> {
+            // redefine column to varchar type
+            handle.execute("drop table if exists videos");
```

Comment:
The test error seems to be due to redefining the table in a transaction while the client has cached data.  Consider using a new table instead of redefining an existing table.